news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -26,96 +27,98 @@ First of all, congratulations to the winners of the April 2017 Spotlights. They
 
-Kokoronashi is yet another great mapset contributed by pishifat. Playing the map feels unique, since it avoids rotational flow and focuses entirely on back-and-forth or linear patterns. You should definitely check this one out if you haven't yet!
+Kokoronashi is yet another great mapset created by **[pishifat](https://osu.ppy.sh/users/3178418)**. The Insane completely avoids circular flow and instead makes use of linear and zig-zag movements, creating a unique gameplay experience. The lower difficulties mirror the concepts of the highest one, so the spread feels very coherent. You should definitely check this one out if you haven't yet!
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/yuki-no-hana.jpg)](https://osu.ppy.sh/s/565432)
 
-A calm, beautiful winter song with a very good-looking skin and a very well designed mapset. While the top difficulty is quite spammy in terms of object density, the visual aspect of the map does not fall apart! Combined with great lower difficulties, this mapset deserves to be featured in the spotlights.
+A calm, beautiful winter song with a very good-looking skin and a very well designed mapset by **[Lan Wings](https://osu.ppy.sh/users/467860)**. While the top difficulty is quite spammy in terms of object density, the visual aspect of the map does not fall apart! Combined with great lower difficulties, this mapset deserves to be featured in the spotlights.
 
 ## osu!taiko
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/classic-pursuit.jpg)](https://osu.ppy.sh/s/373254)
 
-Spooky! Just 7 months after Halloween Hanjamon's Classic Pursuit entered the ranked section, and what a great map it is! Featuring a huge variety of Slider Velocity changes, tricky off-beat patterns, consistency (which does not equal boring) and even 1/8, this map offers a lot of things to enjoy. And if the map is too easy for you: Try it with ANY mod. It's a challenge with Hidden, Hardrock and Doubletime, with the variety of Slider Velocities and complex 1/4+1/8 keeping you on your toes no matter what mod you are using. The set also has nicely done lower diffs, including Hanjamon's signature "Basic" difficulty, which introduces you into the most basic structures of osu!taiko's dons and kats (the red and blue notes!). We hope that you will enjoy this map - and replay it during Halloween!
+Spooky! Just 7 months after Halloween, **[Hanjamon](https://osu.ppy.sh/u/1703330)**'s Classic Pursuit entered the ranked section, and what a great map it is! Featuring a huge variety of Slider Velocity changes, tricky off-beat patterns, consistency (which does not equal boring) and even 1/8, this map offers a lot of things to enjoy. And if the map is too easy for you: Try it with ANY mod. It's a challenge with Hidden, Hardrock and DoubleTime, with the variety of Slider Velocities and complex 1/4+1/8 keeping you on your toes no matter what mod you are using.
```

Comment:
no capitalization after colons unless it's a semicolon, which it shouldn't be here. substitute for a comma instead and exclaim the end of the sentence